alpha/veneer/semver/semver.go

```diff
@@ -114,8 +114,8 @@ func (v Veneer) Render(ctx context.Context) (*declcfg.DeclarativeConfig, error)
 	return &out, nil
 }
 
-func readFile(ref string) (*semverVeneer, error) {
-	data, err := ioutil.ReadFile(ref)
+func readFile(data io.Reader) (*semverVeneer, error) {
```

Comment:
I think `file` is a better name here than `data` at this point in the code, right? Then below, `data` can go back to being `data`, or maybe change to `fileContents` even?  Trying to avoid `data` becoming `d` for sure.

Reply:
I was using it to stay consistent with the name of the field in the Veneer struct, but as long as @grokspawn is okay with it (I don't have enough context to know if data vs. file makes more sense in the api as a whole), I'm happy to change it to `file` everywhere.

Reply:
I don't have that context either, how anything but `data` then, such that `data` can stay `data` below and not become `d`?

Reply:
I can change the `d` to `fileData`. I feel like keeping `data` in the function signature makes it easier to understand if someone is doing a go-def inspection on the function as it matches the Veneer struct that way.

Reply:
that sounds good, really small variable names with no meaning just make me have to go look (and have to remember)

Reply:
So ... I know the last thing that anyone wants is to include someone else who introduces a new opinion, but ... :) 

IMHO, my preference is for 
```
func readFile(reader io.Reader) (*semverVeneer, error) {
```

and 

```
data, err := io.ReadAll(reader)
```